hadoop-ozone/dist/src/main/compose/common/hadoop-test.sh

```diff
@@ -36,9 +33,21 @@ export OZONE_DIR=/opt/ozone
 # shellcheck source=/dev/null
 source "$COMPOSE_DIR/../testlib.sh"
 
-for scheme in o3fs ofs; do
-  execute_robot_test rm -v "SCHEME:${scheme}" -N "hadoop-${HADOOP_VERSION}-hadoopfs-${scheme}" ozonefs/hadoopo3fs.robot
-  execute_robot_test rm -v "SCHEME:${scheme}" -N "hadoop-${HADOOP_VERSION}-mapreduce-${scheme}" mapreduce.robot
+for HADOOP_VERSION in 2.7.3 3.1.2 3.2.2 3.3.1; do
+  export HADOOP_VERSION
+  export HADOOP_MAJOR_VERSION=${HADOOP_VERSION%%.*}
+
+  docker-compose --ansi never --profile hadoop up -d nm rm
```

Comment:
Just to see if I understand correctly, `--profile hadoop` is unnecessary since
```
    profiles:
      - hadoop
```
is already specified in `hadoop.yaml`? I'm fine with keeping it here tho.

Reply:
The definition in `hadoop.yaml` assigns these services to the `hadoop` profile.  Simple `docker-compose up` does not start such services, this way only Ozone is started by `start_docker_env`.  We need `--profile hadoop` to start them at this time.

Reply:
Ah this makes sense. I misunderstood Docker compose profile usage earlier. Thanks.